dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -72,24 +72,24 @@ BEGIN
     POPUP ""
     BEGIN
         MENUITEM SEPARATOR
-        MENUITEM "&Decupează", IDM_CUT
-        MENUITEM "&Copiază", IDM_COPY
-        MENUITEM "&Lipește", IDM_INSERT
+        MENUITEM "&Decupare", IDM_CUT
+        MENUITEM "&Copiere", IDM_COPY
+        MENUITEM "&Lipire", IDM_INSERT
         MENUITEM SEPARATOR
-        MENUITEM "Crează sc&urtătură", IDM_CREATELINK
-        MENUITEM "Șt&erge", IDM_DELETE
-        MENUITEM "&Redenumește", IDM_RENAME
+        MENUITEM "Creare c&omandă rapidă", IDM_CREATELINK
+        MENUITEM "Ş&tergere", IDM_DELETE
+        MENUITEM "Redenu&mire", IDM_RENAME
         MENUITEM SEPARATOR
-        MENUITEM "&Proprietăți", IDM_PROPERTIES
+        MENUITEM "Prop&rietăţi", IDM_PROPERTIES
```

Comment:
Also please a screenshot please that shows how MS does handle the IDM_PROPERTIES for ro-RO.
We discussed that specific one several times, but I guess you are correct with that proposal. **English** has to be moved in that same direction here. I already checked that on my side.

Reply:
![2024-03-27 00_46_37-Resource Hacker - shell32 dll mui](https://github.com/reactos/reactos/assets/113553683/febd2208-432d-4fad-9845-561ee219cef5)

Reply:
Ok "Prop&rietăţi" is hereby confirmed to be correct. That is nice, becaue that is the same that we do aim to use for en-US in the future!